cobalt/browser/application.cc

```diff
@@ -112,6 +113,8 @@ const int64_t kWatchdogTimeInterval = 10000000;
 // violations.
 const int64_t kWatchdogTimeWait = 2000000;
 
+h5vcc::H5vccCrashLog crashLog;
```

Comment:
Don't we need a reference to the existing H5vccCrashLog instance instead of creating a new one?

Reply:
Hmm... maybe not seems new instances will still call the same crash_handler_extension and the CrashLogDictionary is a singleton http://cs/cobalt/cobalt/h5vcc/h5vcc_crash_log.cc;l=112;rcl=0d0cfcbc3f30a3b29d33870429eaaba6a144d708

Reply:
This should not be instantiated here. H5vccCrashLog is a wrappable that means it's associated with a JS instance, which the Application class it not. If you need to re-use functionality that is now in H5vccCrashLog, please split that out in a class that is not wrappable.

Reply:
Changed so `AddCrashLogApplicationState()` uses `CrashLogDictionary` directly without having to instantiate `H5vccCrashLog`.